src/components/masters/program/sidebar/Sidebar.jsx

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { sendTrackEvent } from '@edx/frontend-analytics';
 import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import Links from './Links';
-import linksData from './data/sampleLinks';
 import SidebarBlock from './SidebarBlock';
 
-const Sidebar = () => (
+const Sidebar = ({ programDocuments }) => (
   <>
-    <SidebarBlock title="Program Documents" className="mb-5">
-      <Links id={linksData.id} links={linksData.links} label="program documents" />
-    </SidebarBlock>
+    {programDocuments && programDocuments.display &&
+      <SidebarBlock title={programDocuments.header} className="mb-5">
+        <Links
+          id={programDocuments.header}
```

Comment:
`programDocuments.header` in this case will be something along the line of "Program Documents" and `id` fields can't have spaces in them. So this will probably need to be set programmatically, i.e. `header.toLowerCase().split(' ').join('-')`

Reply:
fixed